hack/common.sh

```diff
@@ -7,9 +7,13 @@ KUBEVIRT_ANSIBLE_DIR="$(
 OUT_DIR=$KUBEVIRT_ANSIBLE_DIR/_out
 TESTS_OUT_DIR=$OUT_DIR/tests
 FUNC_TEST_ARGS="${FUNC_TEST_ARGS:--test.timeout 60m --junit-output=exported-artifacts/tests.junit.xml}"
+RBAC_TEST_ARGS="${RBAC_TEST_ARGS:--test.timeout 60m --junit-output=exported-artifacts/rbac.junit.xml}"
```

Comment:
we don't want to add extra ENV here for some reason, so could you try to research and find a way to reduce `RBAC_TEST_ARGS`?